iceoryx_posh/test/moduletests/test_mepoo_handler.cpp

```diff
@@ -349,3 +349,42 @@ TEST_F(MemoryManager_test, addMemPoolWithChunkCountZeroShouldFail)
     mempoolconf.addMemPool({32, 0});
     EXPECT_DEATH({ sut->configureMemoryManager(mempoolconf, allocator, allocator); }, ".*");
 }
+
+TEST_F(MemoryManager_test, MempoolIndexOutOfRange)
+{
+    constexpr uint32_t ChunkCount{100};
+    mempoolconf.addMemPool({128, ChunkCount});
+    sut->configureMemoryManager(mempoolconf, allocator, allocator);
+
+    std::vector<iox::mepoo::SharedChunk> chunkStore;
+    for (size_t i = 0; i < ChunkCount; i++)
+    {
+        chunkStore.push_back(sut->getChunk(50));
+        EXPECT_THAT(chunkStore.back(), Eq(true));
+    }
+
+    EXPECT_THAT(sut->getMemPoolInfo(100).m_usedChunks, Eq(0));
+}
+
+  TEST_F(MemoryManager_test, GetMemoryConfig)
```

Comment:
Name is misleading. Aren't you testing the `chunkCount` of the mempool config. Why are you testing the `MePooConfig` in a `MemoryManager` test?